src/pydio/res/i18n/it.po

```diff
@@ -1100,17 +1112,23 @@ msgstr "Autenticazione HTTP-Base fallita. Credenziali errate?"
 msgid "Token-based authentication failed, reload credentials?"
 msgstr "Autenticazione basata su Token fallita. Ricaricare le credenziali?"
 
+# Are you sure that is a question? Anyway I translated it into Italian...
 #: sdkremote/exceptions.py:60
 msgid ""
 "Token-based authentication seems unsupported, this may impact performances?"
 msgstr ""
+"L'autenticazione basata su Token sembra non essere supportata, questo può "
+"intaccare le performance?"
 
+# I get an error on parameter %(filesize)
```

Comment:
Which error? Where from? You might need to keep the `%()s` to specify the string cast, I'm not sure though...